spec/views/trips/edit.html.erb_spec.rb

```diff
@@ -6,6 +6,7 @@
     FactoryGirl.create(:wimi, chair: FactoryGirl.create(:chair), user: @user)
     login_as @user
     @trip = assign(:trip, FactoryGirl.create(:trip, user: @user))
+    trip_datespan = FactoryGirl.create(:trip_datespan, trip: @trip)
```

Comment:
Der datespan wird in den tests nicht benutzt, warum hast du den hinzugefuegt? Die datespans sind inzwischen veraltet, im pr #270 ist das refactored